gridlab-d/gridlab-d
# Merge of Feature/1323 - Add transform capability to deltamode

#### What's this Pull Request do?
Resolves the issue of #1323 and allows transforms to tick in deltamode.

Also "made consistent" a couple deltamode time updates

#### Where should the reviewer start?
Pull the code, compile it, run it.  test_deltamode_transforms.glm is a new autotest to explicitly try schedule and player transforms.

#### How should this be tested?
Run autotests - make sure they pass.  Verify the output from test_deltamode_transforms.glm.

#### Any background context you want to provide?
N/A

#### What are the relevant issues?
#1323

#### Screenshots (if appropriate)
N/A

#### Questions:
